eng/common-tests/matrix-generator/pr-matrix-samples/pr-matrix-generation-acceptance.tests.ps1

```diff
@@ -1,11 +1,14 @@
 Import-Module Pester
 
 # Load scenarios before we enter the Describe block so they're available for -ForEach
-$netScenarios = Get-Content (Join-Path $PSScriptRoot net_scenarios.json) | ConvertFrom-Json -AsHashtable
-$pythonScenarios = Get-Content (Join-Path $PSScriptRoot python_scenarios.json) | ConvertFrom-Json -AsHashtable
-$jsScenarios = Get-Content (Join-Path $PSScriptRoot js_scenarios.json) | ConvertFrom-Json -AsHashtable
-$goScenarios = Get-Content (Join-Path $PSScriptRoot go_scenarios.json) | ConvertFrom-Json -AsHashtable
-$javaScenarios = Get-Content (Join-Path $PSScriptRoot java_scenarios.json) | ConvertFrom-Json -AsHashtable
+$script:netScenarios = Get-Content (Join-Path $PSScriptRoot net_scenarios.json) | ConvertFrom-Json -AsHashtable
```

Comment:
These changes enable "recording" of tests so one doesn't have to manually update all expected output JSONs.